test/fullyBackedSortitionPoolTest.js

```diff
@@ -221,6 +234,34 @@ contract("FullyBackedSortitionPool", (accounts) => {
       await pool.selectSetGroup(3, seed, minimumBondableValue, {from: owner})
     })
 
+    it("reverts when operator gets banned in the sortition pool", async () => {
```

Comment:
We could improve the test name a little. We are not failing selection because some operator has been banned and removed from the pool but because of this ban, we no longer have enough unique operators to perform the selection.